gen/internal/tests/thrift/abc-defs.thrift

```diff
@@ -0,0 +1,5 @@
+include "./non_hyphenated.thrift"
```

Comment:
Can we rename this file (and the others)? `abc-defs.thrift` gives no indication of what this file is used for.

Reply:
sure